### PR TITLE
Dockerfile: bump xx to v1.5.0

### PR DESCRIPTION
**Pull request by thaJeztah**
full diff: https://github.com/tonistiigi/xx/compare/v1.4.0...v1.5.0

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 
 ARG GO_VERSION=1.21.10
-ARG XX_VERSION=1.4.0
+ARG XX_VERSION=1.5.0
 ARG OSXCROSS_VERSION=11.3-r7-debian
```
Comment (thaJeztah) [inline Dockerfile:5]: @crazy-max should this one also be updated to a newer version? https://hub.docker.com/r/crazymax/osxcross/tags

**Comment by thaJeztah** [comment]
This one's still failing unfortunately;

```
 > [linux/amd64->arm/v6 base 1/1] RUN xx-apt-get install -y binutils gcc libc6-dev libgcc-10-dev libsecret-1-dev pkg-config:
33.81 distribution that some required packages have not yet been created
33.81 or been moved out of Incoming.
33.83 The following information may help to resolve the situation:
33.83 
33.83 The following packages have unmet dependencies:
34.20  libc6-dev:armel : Depends: linux-libc-dev:armel but it is not going to be installed
34.21  libnsl-dev:armel : Depends: libnsl2:armel (= 1.3.0-2) but it is not installable
34.22  libsepol1-dev:armel : Depends: libsepol1:armel (= 3.1-1) but it is not going to be installed
34.23  libtirpc-dev:armel : Depends: libtirpc3:armel (= 1.3.1-1+deb11u1) but it is not installable
34.31 E: Unable to correct problems, you have held broken packages.
```

**Comment by thaJeztah** [comment]
ah! arm32 packages (`armel`) 🤔

File: Dockerfile (hunk 1 above)
Comment (crazy-max) [reply Dockerfile:5]: Don't think we need to bump to 14.5 yet but `11.3-r8` would be good.

**Review by crazy-max — APPROVED** [review]

**Pull request merged**